client/securedrop_client/logic.py

```diff
@@ -825,7 +830,10 @@ def set_status(self, message: str, duration: int = 5000) -> None:
 
     @login_required
     def _submit_download_job(
-        self, object_type: type[db.Reply] | type[db.Message] | type[db.File], uuid: str
+        self,
+        object_type: type[db.Reply] | type[db.Message] | type[db.File],
+        uuid: str,
+        progress: ProgressProxy | None = None,
```

Comment:
Is this just that we only show file download progress, so sometimes there's no ProgressProxy? If so, makes sense but maybe we can add a quick comment to that effect.

Reply:
Yep, exactly.